Chapter3.html

```diff
@@ -0,0 +1,62 @@
+<html>
+  <head>
+    <title>Lists</title>
+  </head>
+    <body>
```

Comment:
Much better. You can indent body to be at the same spot as the closing head tag.